front/src/modules/rollingStock/components/RollingStockEditor/RollingStockEditorFormHelpers.tsx

```diff
@@ -164,6 +164,7 @@ const RollingStockEditorParameterFormColumn = ({
                 } as SetStateAction<RollingStockParametersValues>);
                 setOptionValue(type.type as string);
               }}
+              limitDecimal={property.title === 'rollingResistanceC' ? 10 : undefined}
```

Comment:
as discussed I think we can have this behavior by default (10 digits max) not just for this one field `rollingResistanceC`